src/actions/tabs.action-creator.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+import { SWITCH_TAB, } from './types';
+
+export function switchTabActionCreator(selectedTab){
```

Comment:
do we really need to be capturing navigation in flux though? the flux store should really only concern itself with data/state. The navigation component should take care of our routing, no?